### PR TITLE
Feat/attestor details select page

### PR DESCRIPTION
**Pull request by rozanagy**

### CODE REVIEW

**Review by Polybius93 — COMMENTED** [review]
Really nice job @rozanagy ! Looks really good! I am leaving some questions, can you take a look on those? Thank you!

File: src/app/components/attestor-details/components/attestor-details-table/attestor-details-table.tsx [hunk 1]
```diff
@@ -1,12 +1,45 @@
...
+import { AttestorDetailsTableItem } from './components/attestor-details-table-item';
 import { AttestorDetailsTableLayout } from './components/attestor-details-table-layout';
 
 export function AttestorDetailsTable(): React.JSX.Element {
   return (
     <AttestorDetailsTableLayout>
-      <AttestorDetailsTableHeader />
-      <AttestorDetailsTableBody />
+      <AttestorDetailsTableHeader
+        children={
```
[the remark this answers is not in the document]
Comment (rozanagy) [reply src/app/components/attestor-details/components/attestor-details-table/attestor-details-table.tsx:13]: actullay vscode marks it as an error if i am not putting the property: children in there.

File: src/app/components/attestor-details/components/attestor-details-table/components/attestor-details-table-item.tsx [hunk 2]
```diff
@@ -14,13 +16,15 @@ interface AttestorDetailsTableItem {
 }
 
 export function AttestorDetailsTableItem(
-  attestorDetailsTableItem: AttestorDetailsTableItem
+  attestorDetailsTableItem: AttestorDetailsTableItemProps
```
[the remark this answers is not in the document]
Comment (rozanagy) [reply src/app/components/attestor-details/components/attestor-details-table/components/attestor-details-table-item.tsx:19]: i used the same method as you used in the protocol-history-table-item.tsx, and i believe your reason was to make the 'if' statement a bit more streamlined and not using a lot of &&.

**Review by Polybius93 — APPROVED** [review]
All good! Nice job!

**Pull request merged**